src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import cn from 'classnames';
+
+interface NotificationProps {
+  error: string | null;
+  onCloseNotifications: () => void;
+}
+
+export const Notification: React.FC<NotificationProps> = ({
+  error,
+  onCloseNotifications,
+}) => {
+  return (
+    <div
+      className={cn(
+        'notification',
+        'is-danger',
+        'is-light',
+        'has-text-weight-normal', {
+          hidden: !error,
+        },
+      )}
+    >
+      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+      <button
```

Comment:
This line has been disabled for ESLint check 'jsx-a11y/control-has-associated-label'. It's better to add an accessible label to the button element for better accessibility.